src/connectors/youtube-music.js

```diff
@@ -23,9 +24,9 @@ Connector.getTrackArt = () => {
 	return null;
 };
 
-Connector.albumSelector = [
-	'.ytmusic-player-bar .yt-formatted-string.style-scope.yt-simple-endpoint[href*="browse/MPREb_"]',
-	'.ytmusic-player-bar .yt-formatted-string.style-scope.yt-simple-endpoint[href*="browse/FEmusic_library_privately_owned_release_detailb_"]',
+const albumSelectors = [
```

Comment:
You still should assign selectors to the `Connector.albumSelector` property. You can assign it right here (as it was before), though I would place it at the top, so it would be like:
```js
const artistSelectors = [
    ...
];
const albumSelectors = [
    ....
];

...

// The base connector will use them to get album info
Connector.albumSelector = albumSelectors;

```